cpp/include/raft/neighbors/detail/ivf_pq_build.cuh

```diff
@@ -16,7 +16,7 @@
 
 #pragma once
 
-#include "ann_utils.cuh"
```

Comment:
Let's move it a bit later together with ivf-flat (I didn't intend to do this much refactoring in this PR to avoid hiding the changes in ivf-pq logic behind all this).